docs/en/stack/ml-biases.asciidoc

```diff
@@ -0,0 +1,96 @@
+[[ml-biases]]
+=== Considering biases in {ml} processes
+
+{ml-cap} is meant to improve our everyday life in a wide palette of activities 
+across the globe. We use {ml} in science, healthcare, business, entertainment, 
+and many other fields. We make decisions based on {ml} results. It is, 
+therefore, crucial to keep in mind the possible biases that can corrupt results, 
+endanger fairness, and jeopardize the success of even the best-intended 
+initiative. Some of these biases can make their way into the {ml} results more 
+easily than we might think.
+
+Various possible biases can negatively influence the result of a {ml} analysis. 
```

Comment:
And add here something of the sort `and affect the people concerned`, otherwise it seems a little bit that the only reason you care about biases is because your ML analysis will be bad, while in reality you need to care about the people being impacted.